README.md

```diff
@@ -72,6 +72,14 @@ Delete the app with
 
 This removes the Serverless app from Twilio. This will ensure that no further cost are incurred by the app.
 
+## Troubleshooting The Twilio CLI
+
+If any errors occur after running a [Twilio CLI RTC Plugin](https://github.com/twilio-labs/plugin-rtc) command, then try the following steps.
+
+1. Run `twilio plugins:update` to update the rtc plugin to the latest version.
+1. Run `twilio rtc:apps:video:delete` to delete any existing authentication servers.
+1. Run `twilio rtc:apps:video:deploy --authentication passcode` to deploy a new authentication server.
```

Comment:
Should be 1, 2, 3 - not 1, 1, 1  😄

Also the last one: 
 3. Run `npm run deploy:twilio-cli` to deploy a new video app.